HoloPipelines/pipelines/components/lungSegment/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Is the complete content of this directory copied from this person's code? Or did you take and modify it significantly? Anyways I feel like we should credit it more clearly, and include a link to their repo (maybe in the README)

Reply:
Credit added in the readme.md (including the link to the original repository)